Flow.Launcher/Storage/UserSelectedRecord.cs

```diff
@@ -14,72 +15,90 @@ public class UserSelectedRecord
         private const int HASH_INITIAL = 23;
 
         [JsonInclude]
-        public Dictionary<int, int> records { get; private set; }
+        public Dictionary<int, int> recordsWithQuery { get; private set; }
+
+        [JsonInclude, JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public Dictionary<string, int> records { get; private set; }
+
 
         public UserSelectedRecord()
         {
-            records = new Dictionary<int, int>();
+            recordsWithQuery = new Dictionary<int, int>();
```

Comment:
Not sure whether this name is great......but we cannot use records as name since the legacy one is using it.